uberfire-extensions/uberfire-metadata/uberfire-metadata-backends/uberfire-metadata-backend-infinispan/src/main/java/org/uberfire/ext/metadata/backend/infinispan/provider/InfinispanContext.java

```diff
@@ -311,24 +312,27 @@ public void addProtobufSchema(String clusterId,
     }
 
     public void loadProtobufSchema(RemoteCache<String, String> metadataCache) {
-        metadataCache.entrySet()
-                .stream()
-                .filter(entry -> !entry.getKey().equals(addCachePrefix(SCHEMA_PROTO)))
-                .forEach((entry) -> {
-                    int index = entry.getKey().lastIndexOf('.');
-                    String protoTypeName = entry.getKey().substring(0,
-                                                                    index);
-                    String proto = entry.getValue();
-
-                    try {
-                        marshaller.registerSchema(protoTypeName,
-                                                  proto,
-                                                  KObject.class);
-                    } catch (IOException e) {
-                        throw new InfinispanException("Can't add protobuf schema <" + protoTypeName + "> to cache",
-                                                      e);
-                    }
-                });
+
+        new Retry(5, () -> {
```

Comment:
What will happened if we run out of retries? IMO there should be some logging or/and exception that loading protobuf schema failed.